README.md

```diff
@@ -25,43 +52,25 @@ import MyComponentThatHasAQuery, {
 
 export default {
   title: 'My Story',
-  decorators: [withApolloClient],
 };
 
 export const example = () => <MyComponentThatHasAQuery />;
 
-example.story = {
-  parameters: {
-    apolloClient: {
-      mocks: [
-        { request: { query: MyQuery }, result: { data: { viewer: null } } },
-      ],
-    },
+example.parameters = {
+  apolloClient: {
+    mocks: [
+      { request: { query: MyQuery }, result: { data: { viewer: null } } },
+    ],
   },
 };
 ```
 
-## Usage in `preview.js`
-
-```js
-import { withApolloClient } from 'storybook-addon-apollo-client';
-import { addDecorator } from '@storybook/react';
-import { InMemoryCache } from 'apollo-cache-inmemory';
-
-const cache = new InMemoryCache();
-
-addDecorator(
-  withApolloClient({
```

Comment:
Would it be useful to include upgrade instructions for anyone already using this with Apollo v3?
My sample app had stories wrapped in `withApolloClient` and was throwing some crazy unexpected errors since that is no longer exported (which means the Story is wrapped in an undefined decorator)

Reply:
Thanks for updating this 👍